pkg/cmd/operator/operator.go

```diff
@@ -201,22 +202,26 @@ func Run(healthPort, monitoringPort int32, leaderElection bool, leaderElectionID
 		}
 	}
 
+	defaultNamespaces := map[string]cache.Config{
+		operatorNamespace: {},
+	}
+
 	options := cache.Options{
-		ByObject: selectors,
```

Comment:
We need to include those selector somewhere for caching reason. It seems this is no longer applied anywhere, or am I missing something?

Reply:
You are right, something got lost here. I will add it again.